hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDFSClientExcludedNodes.java

```diff
@@ -89,6 +89,10 @@ public void testExcludedNodesForgiveness() throws IOException {
     conf.setLong(
         HdfsClientConfigKeys.Write.EXCLUDE_NODES_CACHE_EXPIRY_INTERVAL_KEY,
         2500);
+    // Set min replication for blocks to be written as 1.
+    conf.setInt(
+        HdfsClientConfigKeys.BlockWrite.ReplaceDatanodeOnFailure.MIN_REPLICATION,
+        1);
```

Comment:
Why are we bothering the existing tests? It should pass without any new config change, else it would be an incompatible change for the existing clients